### DISCUSSION TITLE
Guide

### DISCUSSION THREAD

**Discussion by tamathehutt** [AveYo/MediaCreationTool.bat#15] [Q&A]
How do I actually use this software? I am not sure what I am supposed to have as well as the .bat .cmd files.

**Comment by Mikaela** [comment]
You can download the repository (e.g. https://github.com/AveYo/MediaCreationTool.bat/archive/refs/heads/main.zip if you wish to avoid `git clone`) and after extracting it right click `MediaCreationTool.bat` and select "Run as administrator". Wait a moment and you should get a graphical user interface asking what you wish to do and I think it's somewhat self-explanatory.

However you should read through the files to confirm that they aren't doing anything nasty (on principle) before executing them.

**Reply by AveYo** [reply]
you can save the raw script file as well (works fine in repository, unlike in the former gist)  
and you should not run it as administrator, because the script will ask itself when needed
